src/payment/strategies/opy/opy-script-loader.ts

```diff
@@ -0,0 +1,31 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '../../errors';
+
+import { OpyHostWindow, OpyWidgets } from './opy-library';
+
+const AU_REGION_URL = 'https://widgets.openpay.com.au/lib/openpay-widgets.min.js';
+
+const SOURCES = {
+    AU: AU_REGION_URL,
+    NZ: AU_REGION_URL,
+    UK: AU_REGION_URL,
+    US: 'https://widgets.opy.com/lib/openpay-widgets.min.js',
+};
+
+export default class OpyScriptLoader {
+    constructor(
+        private _scriptLoader: ScriptLoader,
+        private _window: OpyHostWindow = window
+    ) { }
+
+    async loadOpyWidgets(region: 'AU' | 'NZ' | 'UK' | 'US' = 'AU'): Promise<OpyWidgets> {
```

Comment:
Just a question why not create a enum above to store these regions, so they have a central place to manage.
https://www.typescriptlang.org/docs/handbook/basic-types.html#enum